### ISSUE TITLE
Control Codes

### ISSUE THREAD

**Issue by EndriTuma** [tonsky/FiraCode#1447]
I think you can fix the control codes
because of this:
https://user-images.githubusercontent.com/110048764/181056662-6425b7c4-87df-4be6-8879-b5136a7b44be.PNG

**Comment by wopian** [comment]
The text in the image is not using the Fira Code font

**Comment by EndriTuma** [comment]
it is using the fira code font

**Comment by EndriTuma** [comment]
maybe the creator of this font forgot to fix it

**Comment by tonsky** [comment]
What are their unicode values?

**Comment by EndriTuma** [comment]
Symbol: ␆  Unicode: U+2406
Symbol:␈   Unicode: U+2408
Symbol:␇   Unicode: U+2407
Symbol:␘   Unicode: U+2418
Symbol:␍   Unicode: U+240d
Symbol:␐  Unicode:  U+2410
Symbol:␡  Unicode:  U+2421
Symbol:␔  Unicode:  U+2414
Symbol: ␓ Unicode:  U+2413
Symbol :␒  Unicode: U+2412
Symbol :␙  Unicode: U+2419
Symbol :␃  Unicode: U+2403
Symbol: ␄  Unicode: U+2404
Symbol: ␗  Unicode: U+2417
Symbol: ␗  Unicode: U+2405
Symbol: ␛  Unicode: U+241b
Symbol: ␜  Unicode: U+241c
Symbol: ␜  Unicode: U+240c

i used control codes which were not in the notepad file because i didnt know there were more control codes than the ones in the notepad file

**Comment by tonsky** [comment]
And where are those used?

**Comment by EndriTuma** [comment]
in the showcases.txt file

**Comment by EndriTuma** [comment]
in the showcases.txt file

**Comment by EndriTuma** [comment]
i figured out the unicodes with vscode with the lua programming language because i did not know the unicodes

**Comment by EndriTuma** [comment]
in the showcases.txt file

**Comment by tonsky** [comment]
No, those are definitely in the font. Maybe Notepad has problems displaying them?

<img width="1002" alt="Screenshot 2022-08-01 at 14 34 08" src="https://user-images.githubusercontent.com/285292/182148709-6373cdbb-f023-4b16-acc5-da5c186b0dc9.png">

**Comment by EndriTuma** [comment]
I Don't Think It Works For Me
